docs/recipes/jspm-systemjs.md

```diff
@@ -1,6 +1,3 @@
-> **Please note, this recipe has not yet been updated for Babel 7 support in AVA 1.0.**
-
----
 
```

Comment:
This line ought to be removed as well.

Reply:
Sounds good to me! I've addressed the requested changes : ^)